test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -223,6 +214,72 @@ TEST(help_add_test, add_option)
                            "SeqAn version: 3.0.0");
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
+}
+
+TEST(test_add_copyright, copyright)
+{
+    std::string stdout;
+    std::string expected;
+
+    // Tests the --copyright call.
+    const char * argvCopyright[] = {"./copyright", "--copyright"};
+    argument_parser copyright("copyright", 2, argvCopyright);
 
-   // EXPECT_THROW(parser6.parse(), parser_interruption);
+    std::ifstream license_file{std::string{{SEQAN_INCLUDE_DIR}} + "/../LICENSE"};
+    std::string license_string{(std::istreambuf_iterator<char>(license_file)),
+                                std::istreambuf_iterator<char>()};
+    // Test --copyright with empty short and long copyright info.
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright copyright information not available."
+                               "================================================================================"
+                               "This program contains SeqAn3 code licensed under the following terms:"
+                               "--------------------------------------------------------------------------------"
+                               + license_string);
+        EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
+                                   expected | ranges::view::remove_if(is_space)));
+    }
+    // Test --copyright with a non-empty short copyright and an empty long copyright.
+    copyright.info.short_copyright = "short copyright\nshortcopyright";
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright full copyright information not available. Displaying short copyright information instead:"
+                               "short copyright"
+                               "short copyright"
```

Comment:
double short copy right ? ah I see that this is intentional. can you change it to 
`short copyright line 1\n short copy right line 2`? otherwise people (like me) might think this is an error.